### PR TITLE
Reflection padding

### PR DESCRIPTION
**Pull request by Keloo**
Adding reflection padding before processing image and the cropping it out.

Before:
jessica_wave_original
After:
jessica_wave

### CODE REVIEW

**Comment by Zueuk** [comment]
this is useful, but for me BORDER_SIZE = 30 turned out to be not enough, had to increase it to at least 40

**Comment by Keloo** [comment]
Good point, I'll take a look and push the changes soon. 
Thx

**Comment by dovanchan** [comment]
@Keloo  Hi,can you tell me what parameters you used in training?(or default?)
content weight 1.5e1?

**Comment by Keloo** [comment]
@dovanchan Default parameters

**Comment by dovanchan** [comment]
Hi,Can you show me your style.py 、 evaluate.py and transform_video.py  ,This is mine.
image
image
image
@Keloo

**Comment by Keloo** [comment]
@dovanchan I published all my changes in the commit (pull request) "Reflection padding" 
https://github.com/lengstrom/fast-style-transfer/pull/56/commits/98992e43a879eb8267862742e29f4afd84b9080c

I used the default library and just added a border with reflection padding then I just cropped it out (after the style transfer) so the border effect is out.

**Comment by dovanchan** [comment]
Hi,Can you sent me your style.py 、 evaluate.py and transform_video.py to my email( wodovan@gmail.com) Thank you~
because I found the default setting in this code have ever change;
such as the content weight;In the command recommend ,it is 1.5e1
but in style.py,It 's 7.5.So I wanna know what are you using

**Comment by Keloo** [comment]
@dovanchan ok, it make sense if they changed the settings. I'll send you an email soon :)

**Comment by dovanchan** [comment]
Yes,I just wanna the file and the command you use which you get these beautiful result;thank you~

**Comment by dovanchan** [comment]
By the way,can you give me the content so that I can have a test for the parameters~

**Comment by Keloo** [comment]
@dovanchan I've sent you an email

**Comment by dovanchan** [comment]
Really Thank Again!~

File: src/utils.py [hunk 1]
```diff
@@ -20,6 +22,15 @@ def get_img(src, img_size=False):
        img = scipy.misc.imresize(img, img_size)
    return img
 
+def get_img_bordered(src, img_size=False, border_size = BORDER_SIZE):
```
Comment (ankur2136) [inline src/utils.py]: img_size should not be a boolean. 
If you want no default value use None.

Comment (Keloo) [inline src/utils.py]: thx for suggestion, I submitted a fix :)

**Pull request closed without merging**